tests/evm_script_factories/test_top_up_allowed_recipients.py

```diff
@@ -263,27 +352,35 @@ def test_fail_create_evm_script_if_sum_exceeds_limit(
         set_limit_role_holder,
         _,
     ) = allowed_recipients_registry
+    (tokens_registry, _, add_token_role_holder, _) = allowed_tokens_registry
 
-    registry.addRecipient(recipients[0], "Test Recipient 1", {"from": add_recipient_role_holder})
-    registry.addRecipient(recipients[1], "Test Recipient 2", {"from": add_recipient_role_holder})
+    registry.addRecipient(
+        recipients[0], "Test Recipient 1", {"from": add_recipient_role_holder}
+    )
+    registry.addRecipient(
+        recipients[1], "Test Recipient 2", {"from": add_recipient_role_holder}
+    )
     registry.setLimitParameters(int(20e18), 12, {"from": set_limit_role_holder})
 
     top_up_factory = owner.deploy(
-        TopUpAllowedRecipients, owner, registry, finance, ldo, easy_track
+        TopUpAllowedRecipients, owner, registry, tokens_registry, finance, easy_track
     )
 
+    tokens_registry.addToken(ldo, {"from": add_token_role_holder})
+
     with reverts("SUM_EXCEEDS_SPENDABLE_BALANCE"):
         top_up_factory.createEVMScript(owner, call_data)
 
 
 def test_create_evm_script_correctly(
```

Comment:
It's better to use a token with decimals that differ from the recipients registry. To validate that normalization also happens